### ISSUE TITLE
Enable to change the LLM parameters at runtime

### ISSUE THREAD

**Issue by andreadimaio** [quarkiverse/quarkus-langchain4j#757]
The idea is to understand if there is a way to change the parameters passed to the LLM at runtime.
Today the prompt can be changed using `@UserMessage` and `@V`:
```java
public String poem(@UserMessage String prompt, @V("topic") String topic)
```
but the parameters are fixed to what we have in `application.properties`.

**Comment by geoand** [comment]
By parameters, you mean the model specific parameters?

**Comment by andreadimaio** [comment]
Yep.

**Comment by geoand** [comment]
I think @langchain4j has something for that, but I can't find the issue now

**Comment by andreadimaio** [comment]
For example, suppose there is an API that returns something like this:
```json
"prompt": {
        "model_id": "meta-llama/llama-3-70b-instruct",
        "model_parameters": {
            "decoding_method": "greedy",
            "max_new_tokens": 400,
            "min_new_tokens": 0,
            "stop_sequences": [],
            "repetition_penalty": 1
        },
        "input": "my super prompt"   
    }
```

The idea is to use what I have in the returned object.

**Comment by geoand** [comment]
Right, I remember there was some proposal in @langchain4j but I can't find it now...

**Comment by andreadimaio** [comment]
I found this that seems to be what I'm looking for  langchain4j/langchain4j#1261

**Comment by langchain4j** [comment]
Hi, https://github.com/langchain4j/langchain4j/pull/1261 is for the low-level `ChatLanguageModel` API, there is nothing yet for AI Services

**Comment by andreadimaio** [comment]
With the new version of langchain4j it is possible to use the [ChatRequest](https://github.com/langchain4j/langchain4j/blob/cdf818c0d9f18a3aec4e96bc2eecce2cc1342ca9/langchain4j-core/src/main/java/dev/langchain4j/model/chat/ChatLanguageModel.java#L50) object

**Comment by geoand** [comment]
👌
